arch/arm/mach-msm/lge/board-8974-rfkill.c

```diff
@@ -190,6 +190,9 @@ static int bluetooth_set_power(void *data, bool blocked)
 		gpio_direction_output(GPIO_BT_RESET_N, 0);
 		printk(KERN_ERR "Bluetooth RESET LOW!!");
 	}
+#ifdef CONFIG_BCM4335BT
```

Comment:
Add a new line before this. Just for readability.

Reply:
Done!